### PR TITLE
Updated operator doc

### PR DESCRIPTION
**Pull request by fbolton**
Merge request superseding Paul's PR https://github.com/debezium/debezium/pull/1185.
Merge conflicts have been resolved in this branch.

### CODE REVIEW

**Comment by Naros** [comment]
I do want to note that in PR https://github.com/debezium/debezium/pull/1216, I looked at the table styles mentioned by @PaulRMellor and reduced the usage of `options="header,footer"` as well as the `role="..."` parts of the table declarations.  I'm sure there are likely other aspects that could be aligned to fit your doc process, but I've left those as-is.  @fbolton @PaulRMellor, if you guys have any other suggestions, feel free to let me know the specifics as I don't want to stumble on your efforts.

**Comment by gunnarmorling** [comment]
@Naros, sounds great! Thanks!

File: documentation/modules/ROOT/pages/connectors/mongodb.adoc [hunk 1]
```diff
@@ -7,112 +9,163 @@ include::../_attributes.adoc[]
...
+This new server becomes a secondary (and able to handle queries) when it catches up to the tail of the primary's oplog.
 
-The Debezium MongoDB connector uses this same replication mechanism, though it does not actually become a member of the replica set. Just like MongoDB secondaries, however, the connector always reads the oplog of the replica set's primary. And, when the connector sees a replica set for the first time, it looks at the oplog to get the last recorded transaction and then performs an _intial sync_ of the primary's databases and collections. When all the data is copied, the connector then starts reading the oplog from the position it read earlier. Operations in the MongoDB oplog are https://docs.mongodb.com/manual/core/replica-set-oplog/[idempotent], so no matter how many times the operations are applied, they result in the same end state.
+The MongoDB connector uses this same replication mechanism, though it does not actually become a member of the replica set.
```
Comment (gunnarmorling) [inline documentation/modules/ROOT/pages/connectors/mongodb.adoc]: It's a bit unfortunate that this commit does a few things at once (which all are good by themselves). Would have been nice if the wrapping of sentences to new lines had been its own commit (which then can be largely ignored while reviewing), and doing other more critical changes in a separate one (which then is easier to review).

Comment (fbolton) [reply documentation/modules/ROOT/pages/connectors/mongodb.adoc]: Ah, yes, this is really very awkward to review. Sorry about that! The initial commit from Paul captures multiple phases of work on the connector content, so there is really no practical way to separate these different phases now. But I have made a recommendation in the contributor notes (work in progress), to recommend that contributor's avoid this in future.

**Comment by fbolton** [comment]
@naros We probably need to have a discussion about table formats to sort out some recommendations. I don't really have all of the issues clear in my head right now, so it would be good to chat about it and capture some recommendations for the contributor's guide.

File: documentation/modules/ROOT/pages/connectors/mongodb.adoc (hunk 1 above)
Comment (gunnarmorling) [reply documentation/modules/ROOT/pages/connectors/mongodb.adoc]: Yes, would be really good to stick to this rule, in particular as we have been talking about this before. Thanks!

**Comment by gunnarmorling** [comment]
Closing in favour of #1217. @fbolton, just in case you're not aware of this: you can push further commits to a PR's branch while it's open, instead of creating a new one. Of course that still may be preferable in some cases, e.g. when a later commit largely changes course over the already existing one. But e.g. here, where #1217 simply adds the DOCUMENTATION.md update, amending the existing PR would be totally fine.

**Comment by gunnarmorling** [comment]
> it would be good to chat about it and capture some recommendations for the contributor's guide.

+1 Could the two of you sync on this and come up with something?

**Comment by fbolton** [comment]
@gunnarmorling I actually intended for the `DOCUMENTATION.md` to be a separate contribution, but I accidentally branched from fbolton:updated-operator-doc when creating the #1217 PR. Thanks for catching this and sorting out the mess!

**Comment by Naros** [comment]
> +1 Could the two of you sync on this and come up with something?

Absolutely, I'll sync up with @fbolton via email on this.

**Pull request closed without merging**